client/components/course/Outline/index.vue

```diff
@@ -36,22 +36,27 @@
 </template>
 
 <script>
-import { mapGetters, mapMutations } from 'vuex-module';
+import { mapActions, mapGetters, mapMutations } from 'vuex-module';
 import Activity from './Activity';
+import api from '../../../api/activeUsers';
 import Draggable from 'vuedraggable';
 import filter from 'lodash/filter';
 import find from 'lodash/find';
 import map from 'lodash/map';
 import NoActivities from './NoActivities';
+import { pingInterval } from '../../../../config/shared/activeUsers';
```

Comment:
There is an alias for [this](https://github.com/ExtensionEngine/tailor/blob/835bc999488c5e376d6c1ac814aa1b874b6f353c/poi.config.js#L19)